scripts/measures/enrich-measures-data.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const path = require('path');
+
+const aciRelations = require('../../util/measures/aci-measure-relations.json');
+const cpcPlusGroups = require('../../util/measures/cpc+-measure-groups.json');
+
+const qpp = fs.readFileSync(path.join(__dirname, '../../measures/measures-data.json'), 'utf8');
+fs.writeFileSync(path.join(__dirname, '../../measures/measures-data.json'), enrichMeasures(JSON.parse(qpp)));
+
+function enrichMeasures(measures) {
+  enrichACIMeasures(measures);
+  enrichCPCPlusMeasures(measures);
+  return JSON.stringify(measures, null, 2);
+};
+
+/**
+ * Will add extra metadata to ACI measure that are not directly available
+ * in machine inferable format at https://qpp.cms.gov/api/v1/aci_measures
+ * After this function executes, an ACI measure will have reporting category and substitutes.
+ *  - substitutes: contains other measures that surrogates of a given measure.
+ *  - reportingCategory: corresponds to the measure performance category
+ * @param measures - the measures to enrich
+ */
+function enrichACIMeasures(measures) {
+  // add extra ACI metadata to ACI measure
+  measures
+    .filter(measure => measure.category === 'aci')
+    .forEach(measure => {
+      // find the relation and populate reporting category and substitutions
+      var aciRelation = aciRelations[measure.measureId];
+      if (aciRelation) {
+        measure.reportingCategory = aciRelation.reportingCategory;
+        measure.substitutes = aciRelation.substitutes;
+      }
+    });
+}
+
+/**
+ * Will add extra metadata to CPC+ measures
+ * @param {array} measures
+ */
+function enrichCPCPlusMeasures(measures) {
+  measures
+    .filter(measure => measure.category === 'quality')
+    .forEach(measure => {
+      Object.keys(cpcPlusGroups).forEach((groupId) => {
+        var match = cpcPlusGroups[groupId].find((id) => id === measure.eMeasureId);
+        if (match !== undefined) {
+          measure.cpcPlusGroup = groupId;
+        }
+      });
+    });
+};
```

Comment:
Why do these functions need to be in this new file when the same code is in `merge-measures-data.js`?

Reply:
Actually, I think `merge-measures-data.js` is deprecated.  I will verify and move it into the `archive` folder.

Reply:
@rockykitamura could we remove the new functions from merge-measures-data so there's no confusion about what script should own these functions?

Reply:
No problem.  I've removed the functions from the merge-measures-data script.